int32 id
int32 type
int32 n_points
float64 distance
geometry_msgs/Point[] points # x,y: coordinate, z: distance to goal